taxi/tests/test_models.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car
+
+
+class ModelsTest(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(name="Test", country="test")
+        self.assertEqual(
+            str(manufacturer),
+            f"{manufacturer.name} {manufacturer.country}")
+
+    def test_car_str(self):
+        name = "Test"
+        country = "Test"
+        model = "Test"

```

Comment:
why did you decide to create these variables? Also, you have the same value for all of them, so you can create only `test_data="Test"` and use it everywhere